prow/plugins/trickortreat/trickortreat.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package trickortreat adds halloween images to an issue or PR in response to a /trick-or-treat comment
+package trickortreat
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+	"regexp"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+var (
+	match        = regexp.MustCompile(`(?mi)^/trick(\-)?or(\-)?treat(?: (.+))?\s*$`)
+	trickortreat = &realSnicker{}
+
+	// A curated list that contains safe images URLs that we feel confident to share.
+	// TODO: use an API for obtaining list of images, for example from https://www.pexels.com/api/documentation.
+	candiesImgs = []string{
```

Comment:
Will you please at least put these in another file?

Reply:
Done